templates/csi-daemonset.yaml

```
@@ -43,6 +43,7 @@ spec:
       {{- if .Values.csi.priorityClassName }}
       priorityClassName: {{ .Values.csi.priorityClassName }}
       {{- end }}
+      hostNetwork: {{ default false .Values.hostNetwork }}
```

Can you set the default in values.yaml, similar to [server.hostNetwork](https://github.com/shuyanl-qualtrics/vault-helm/blob/36f9fcd750c7d9b348639d31b3f278cd7e4641bb/values.yaml#L1026) and [injector.hostNetwork](https://github.com/shuyanl-qualtrics/vault-helm/blob/36f9fcd750c7d9b348639d31b3f278cd7e4641bb/values.yaml#L330)?

```suggestion
      hostNetwork: {{ .Values.csi.hostNetwork }}
```